### PR TITLE
hook jenkins job for aws-actuator

### PR DESCRIPTION
**Pull request by paulfantom**
Allow prow to trigger jenkins job from https://github.com/openshift/aos-cd-jobs/pull/1554

@stevekuznetsov PTAL

### CODE REVIEW

**Comment by petr-muller** [comment]
I think you are missing one layer between the `presubmits` and the actual list of jobs, it should look like like this (I derived you want to set it up for openshift/cluster-api-provider-aws from the path in `ci-operator/jobs/...`)

```
presubmits:
  openshift/cluster-api-provider-aws:
  - name: ci-kubernetes-aws-actuator
  ...
```

File: cluster/ci/config/prow/config.yaml [hunk 1]
```diff
@@ -83,6 +83,7 @@ tide:
     - openshift/machine-config-operator
     - openshift/cluster-ingress-operator
     - openshift/cluster-dns-operator
+    - openshift/cluster-api-provider-aws
     labels:
     - lgtm
     - approved
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/config.yaml]: If you want to require approval, you need `OWNERS` files and the `approve` plugin

Comment (paulfantom) [reply cluster/ci/config/prow/config.yaml]: there is OWNERS in cluster-api-provider-aws repo, do I also need one here?

**Comment by paulfantom** [comment]
Can we merge it?

/assign @stevekuznetsov

File: cluster/ci/config/prow/plugins.yaml [hunk 2]
```diff
@@ -433,3 +433,7 @@ plugins:
   - trigger
   - approve
   - blunderbuss
+
+  openshift/cluster-api-provider-aws:
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/plugins.yaml]: You won't need `trigger` here after #1229

**Comment by paulfantom** [comment]
Resolved again :)

**Pull request merged**